corehq/apps/app_manager/management/commands/audit_app_profile_properties.py

```diff
@@ -0,0 +1,40 @@
+import logging
+
+from django.core.management.base import BaseCommand
+
+from corehq.apps.domain.models import Domain
+from corehq.apps.app_manager.dbaccessors import get_apps_in_domain
+from corehq.apps.app_manager.commcare_settings import get_custom_commcare_settings
+
+logger = logging.getLogger('audit_app_profile_properties')
+logger.setLevel('DEBUG')
+
+
+class Command(BaseCommand):
+    help = '''
+        Determine how many apps have changed their profile properties from the default values.
+    '''
+
+    def handle(self, *args, **options):
+        domains = [row['key'] for row in Domain.get_all(include_docs=False)]
+        settings = {s['id']: s['default'] for s in get_custom_commcare_settings() if 'default' in s}
+        deviant_counts = {id: 0 for id in settings.keys()}
+        app_count = 0
+        for domain in domains:
+            for app in get_apps_in_domain(domain, include_remote=False):
+                #logger.info("looking at app {}".format(app.id))
+                if ('properties' in app.profile):
+                    app_count = app_count + 1
+                    for id, default in settings.iteritems():
+                        if (id not in app.profile['properties']):
+                            #logger.info("{}: not found".format(id))
+                            pass
+                        elif (app.profile['properties'][id] != default):
```

Comment:
I imagine you developed this through debugging the output, given the commented logging statements, but could reduce this three-clause-if down to one:
```py
# somewhere outside the loop
NOTHING = object()  # None-like value that != None in case default is ever None
...
# inside the loop
if app.profile['properties'].get(id, NOTHING) != default:
    ...